### PR TITLE
chore: bump vyper version

### PR DESCRIPTION
**Pull request by pandadefi**
### What I did

Update vyper version to 0.2.13

Related issue: #

### Checklist

- [ ] I have confirmed that my PR passes all linting checks
- [ ] I have included test cases
- [ ] I have updated the documentation
- [ ] I have added an entry to the changelog

### CODE REVIEW

**Comment by iamdefinitelyahuman** [comment]
better to do `v0.2.14` :)

**Comment by pandadefi** [comment]
> better to do `v0.2.14` :)

done

File: tests/network/transaction/test_subcalls.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 
 vyper_source = """
-# @version 0.2.12
+# @version 0.2.13
```
Comment (iamdefinitelyahuman) [inline tests/network/transaction/test_subcalls.py:27]:
```suggestion
# @version 0.2.14
```

Comment (pandadefi) [reply tests/network/transaction/test_subcalls.py]: Search and replace missed this one 😔

**Pull request closed without merging**